awx/main/models/notifications.py

```diff
@@ -383,12 +383,12 @@ def context(self, serialized_job):
         and a url to the job run."""
         job_context = {'host_status_counts': {}}
         summary = None
-        if hasattr(self, 'job_host_summaries'):
-            summary = self.job_host_summaries.first()
-        if summary:
-            from awx.api.serializers import JobHostSummarySerializer
-            summary_data = JobHostSummarySerializer(summary).to_representation(summary)
-            job_context['host_status_counts'] = summary_data
+        if getattr(self, f'{self.model_to_str()}_events'):
```

Comment:
I don't think this if condition is needed. The `UnifiedJob` class has the `get_event_queryset()` function defined so all models that inherit from `UnifiedJob` will have this property.

Reply:
yeah sure make sense, we can remove the if condition.